packages/dd-trace/src/exporters/span-stats/writer.js

```diff
@@ -0,0 +1,55 @@
+
+const { SpanStatsEncoder } = require('../../encode/span-stats')
+
+const pkg = require('../../../../../package.json')
+
+const BaseWriter = require('../common/writer')
+const request = require('../common/request')
+const log = require('../../log')
+
+class Writer extends BaseWriter {
+  constructor ({ url }) {
+    super(...arguments)
+    this._url = url
+    this._encoder = new SpanStatsEncoder(this)
+  }
+
+  _sendPayload (data, _, done) {
+    makeRequest(data, this._url, (err, res) => {
+      if (err) {
+        log.error(err)
+        done()
+        return
+      }
+      log.debug(`Response from the intake: ${res}`)
+      done()
+    })
+  }
+}
+
+function makeRequest (data, url, cb) {
+  const options = {
+    path: '/v0.6/stats',
```

Comment:
Stats didn't exist before 0.6, as far as I'm aware. It would not be supported. As for agentless, stats are not supported there. We would need entirely different encoding for that as the agent -> backend part of stats is protobuf not msgpack.

Reply:
> Stats didn't exist before 0.6, as far as I'm aware. It would not be supported.

That's basically my question. Since it's not supported, are we just going to be sending data and fail every time? It seems to me that similar to my other concern below, we should rely on the endpoint version here so that we can fallback.

Reply:
That's why it's behind a flag. It's not supposed to be enabled unless the user says to do so.